### PR TITLE
add controllers

### PR DESCRIPTION
**Pull request by BANCUNGUYE66**
### create controllers and views for app

For each URL I created:

- [x] A route.
- [x] An action in the correct controller.
- [x] A view file.

### CODE REVIEW

File: app/controllers/navigation_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  end
+
+  def resume
+    # Placeholder code for resume action
+  end
+
+  def refresh
+    # Placeholder code for refresh action
+  end
+end
```
Comment (omar-muhamad) [inline app/controllers/navigation_controller.rb]: - [x] As per project [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#project-requirements), as per the this image:

image

Please delete what you did here and create 2 controllers with 4 methods, 4 views, and 4 routes one for each method:
- Users controller with index and show methods.
- Posts controller with index and show methods.
don't forget to add a view and route for each method and also don't use `rails generate scaffold` command as you did with the navigation controller.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @BANCUNGUYE66,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @BANCUNGUYE66 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ All linters checks are ok
✅ The correct workflow is used. 
✅ Very descriptive Pull Request(PR)
✅ Your work is well documented.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  # Users routes
+  get '/users', to: 'users#index'
+  get '/users/:id', to: 'users#show', as: 'user'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  # Posts routes
+  get '/posts', to: 'posts#index'
+  get '/posts/:id', to: 'posts#show', as: 'post'
```
Comment (levy002) [inline config/routes.rb:8]: - [OPTIONAL] Hey! You can simplify your routes code like this:

```ruby
resources :users, only: [:index, :show] do
  resources :posts, only: [:index, :show]
end
```

By using the `resources` method with the appropriate options, you can achieve the same routing behavior more concisely. This helps keep your codebase clean and easy to manage. Keep up the good work! 🚀👨‍💻

**Review by levy002 — COMMENTED** [review]
## Hello @BANCUNGUYE66 

Your project is complete!  There is nothing else to say other than... congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**